backend/nomad_connection.go

```diff
@@ -641,18 +641,25 @@ func (c *NomadConnection) watchClientStats(action Action) {
 	}
 }
 
+func nodeUrl(params map[string]interface{}) string {
```

Comment:
is there any case / error condition where params[] could be without addr or especially secure?